articles/mysql/concepts-connectivity-architecture.md

```diff
@@ -68,6 +68,7 @@ The following table lists the gateway IP addresses of the Azure Database for MyS
 | South Africa West	| 102.133.24.0	 | | |
 | South Central US |104.214.16.39, 20.45.120.0  |13.66.62.124  |23.98.162.75 |
 | South East Asia | 40.78.233.2, 23.98.80.12	 | 104.43.15.0 | |
+| Switzerland North | 
```

Comment:
Aravind, do you want to add the IP address too?

Reply:
Okay I will ad the IP and also check if there are other regions missing in this document, thanks Aravind